docs/en/guides/pipeline-integrations/development/sgtk-developer-bootstrapping.md

```diff
@@ -0,0 +1,409 @@
+
+### Requirements
+
+- An understanding of Python programming fundamentals. 
+- A project with an advanced configuration. If you haven't set up a configuration before you can follow the ["Getting started with configurations"](../getting-started/advanced_config.md) guide.
+
+### Steps
+
+1. [Importing the sgtk API for bootstrapping](#part-1---importing-the-sgtk-api-for-bootstrapping)
+2. [Logging](#part-2---logging)
+3. [Authentication](#part-3---authentication)
+4. [Bootstrapping an engine](#part-4---bootstrapping-an-engine)
+5. [launching an app](#part-5---launching-an-app)
+6. [Complete script](#part-6---the-complete-script)
+
+## Part 1 - Importing the sgtk API for bootstrapping
+
+### Where should I import sgtk from?
+
+If you've followed the ["generating a path and publishing it"](sgtk-developer-generating-path-and-publish.md) 
+guide then you'll have covered the step of importing sgtk. In that guide, it states that you must import the sgtk API package from
+the project configuration you wish to work with. This is still true, however, with bootstrapping, it doesn't matter
+which initial sgtk API you import, as any sgtk API can perform the bootstrap operation into a different project configuration.
+As part of this process, it will swap out the currently imported sgtk package for the new project config's sgtk API.
+
+### Downloading a standalone sgtk core API
+
+To start with you need to import an sgtk API package. 
+You could import one from an existing project, however, this might be tricky to conveniently locate.
+A recommended approach would be to download a standalone copy 
+of the [latest core API]((https://github.com/shotgunsoftware/tk-core/releases)) which will be used purely for the purpose of bootstrapping.
+You should store it in a convenient place where it can be imported. 
+If it's not located in a standard location that python will look in, then you can add the path to `sys.path`. 
+
+### Code
+
+```python
+# If your sgtk package is not located in a location where Python will automatically look
+# then add the path to sys.path
+import sys
+sys.path.insert(0, "/path/to/sgtk-package")
+
+import sgtk
+```
+
+## Part 2 - Logging
+
+If you are running this script via an IDE or shell, then you will most likely want to enable the logging
+to be output. To do this you need to run [`LogManager().initialize_custom_handler()`](https://developer.shotgunsoftware.com/tk-core/utils.html#sgtk.log.LogManager.initialize_custom_handler).
+You don't need to provide a custom handler for this purpose, as not providing one will set up a standard stream-based logging handler.
+
+Optionally you can also set the [`LogManager().global_debug = True`](https://developer.shotgunsoftware.com/tk-core/utils.html#sgtk.log.LogManager.global_debug) to give you more verbose output. In other words any
+ `logger.debug()` calls should also be output. You should only set this when developing as it can have an impact on performance. 
+
```

Comment:
I have two links that point to the logging documentation, I'm not sure there is anything more I can link to?

> as opposed to what when it's false?

I've changed it to:
> This means that any `logger.debug()` calls in our code or yours will now be output.
Logging can have an impact on performance, so you should only enable debug logging when developing, and try to limit the amount of `logger.info()` method calls to those that are important to have visibility over during normal operation.

Does that help?
It would be as opposed to it just not being output.